scripts/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+install(DIRECTORY
+  cli
+  DESTINATION "share/codecompass"
+  USE_SOURCE_PERMISSIONS
+  FILES_MATCHING PATTERN "[^.]*")
+
+install(FILES
+  CodeCompass
+  setldlogenv.sh
```

Comment:
Ok, I put it under the `share/codecompass` directory. (The script files also placed here under the cli directory)